private/bufpkg/bufconfig/lint_config.go

```diff
@@ -34,7 +34,7 @@ var (
 		false,
 		false,
 		"",
-		false,
+		true, // We default to allowing comment ignores in v2
```

Comment:
@pkwarren caught an issue with the default `v2` lint config, which should default to allowing comment ignores (https://buf.build/docs/configuration/v2/buf-yaml)

>       # v1 configurations had an allow_comment_ignores option to opt-in to comment ignores.
>       #
>       # In v2, we allow comment ignores by default, and allow opt-out from comment ignores
>       # with the disallow_comment_ignores option.
>       disallow_comment_ignores: false